packages/main/src/plugin/kubernetes/contexts-manager.ts

```diff
@@ -976,6 +976,25 @@ export class ContextsManager {
     return [];
   }
 
+  public async refreshContextState(contextName: string): Promise<void> {
```

Comment:
please add some documentation above, this seems to be the entrypoint of the logic